src/components/SignUp/Group/index.tsx

```diff
@@ -0,0 +1,244 @@
+import { createUserWithEmailAndPassword } from 'firebase/auth';
+import { addDoc, collection } from 'firebase/firestore';
+import React, { useState } from 'react';
+import { Col, Row } from 'react-bootstrap';
+import { Step, useForm, useStep } from 'react-hooks-helper';
+import { useFirebaseContext } from '../../../context/FirebaseContext';
+import { useProfileContext } from '../../../context/ProfileContext';
+import Profile from '../../../pages/Profile';
+import PageContainer from '../../layout/PageContainer';
+import ActorInfo1 from '../Individual/ActorInfo1';
+import ActorInfo2 from '../Individual/ActorInfo2';
+import IndividualBasics from '../Individual/Basics';
+import Demographics from '../Individual/Demographics';
+import OffstageRoles from '../Individual/OffstageRoles';
+import ProfilePhoto from '../Individual/ProfilePhoto';
+import Privacy from '../Privacy';
+import SignUpFooter from '../SignUpFooter';
+
+const GroupSignUp: React.FC<{
+  currentStep: number;
+  setCurrentStep: (x: number) => void;
+}> = ({ currentStep, setCurrentStep }) => {
+  const { firebaseAuth, firebaseFirestore } = useFirebaseContext();
+  const { setAccountRef, setProfileRef } = useProfileContext();
+  const [formData, setForm] = useForm(defaultData); // useForm is an extension of React hooks to manage form state
+
+  // default state for form validation error states per step
+  const [stepErrors, setStepErrors] = useState(
+    createDefaultStepErrorsObj(flatSteps(defaultSteps))
+  );
+
+  // console.log('formData', { formData });
+  // defaults for our defaultSteps
+  const { step, index, navigation } = useStep({
+    initialStep: currentStep,
+    steps: defaultSteps
+  });
+  const stepId = (step as Step).id;
+  // we are just manually assuming they've agreed to this right now
+  // remove once we clean this up
+  const setPrivacyAgree = () => {
+    const target = {
+      name: 'privacyAgreement',
+      value: true
+    };
+    setForm({ target });
+  };
+
+  // submit basics to Firebase, get response, set session
+  const submitBasics = async () => {
+    // we only get here if they've agreed to the privacy agreement
+    setPrivacyAgree();
+
+    const {
+      basicsFirstName,
+      basicsLastName,
+      basicsEmailAddress,
+      basicsPassword,
+      basics18Plus
+    } = formData;
+
+    await createUserWithEmailAndPassword(
+      firebaseAuth,
+      basicsEmailAddress,
+      basicsPassword
+    )
+      .then(async res => {
+        try {
+          const userId = res.user.uid;
+
+          const accountRef = await addDoc(
+            collection(firebaseFirestore, 'accounts'),
+            {
+              basics_18_plus: basics18Plus,
```

Comment:
Does the theater sign-up profile creation include these same fields, like 18+?

Reply:
It won't. This was just an initial stub to get it to where it was before the split.